MIP/mip-2/README.md

```diff
@@ -0,0 +1,87 @@
+
+## Motivation
+
+There is an immediate need to deploy the Movement Token (MOVEToken) using a robust, secure, and auditable upgrade mechanism. By creating two multisigs—one for proposal and one for execution—we ensure a transparent and decentralized governance process over upgrades to critical contracts.
+
+The first multisig allows Movement Labs engineers to propose upgrades via Safe’s multisig service. A remote key will automate the population of upgrade proposals. The second multisig, held by the Movement Foundation, will be responsible for executing the scheduled upgrades after a time delay, ensuring community trust and review.
+
+## Specification
+
+### Multisig Setup
+
+1. **Movement Labs Multisig** (4/5 signatures required):  
+   - Role: Proposing upgrades.
+   - Addresses: Deployed on Ethereum, Sepolia, and Holesky (if available).
+   - Signers: Andy Bell, Richard Melkonian, Liam Monninger, Primata (Engineers involved with Token), and a remote key for automated transaction proposals via SafeKit API's [proposeTransaction](https://docs.safe.global/sdk/api-kit#propose-a-transaction-to-the-service).
+
+2. **Movement Foundation Multisig** (3/5 signatures required, triggerable after 2-day timelock):  
+   - Role: Executing upgrades.
+   - Addresses: Deployed on Ethereum, Sepolia, and Holesky (if available).
+   - Signers: TBD.
+
+Both multisigs are part of a timelock mechanism to enforce a 2-day waiting period between the proposal and execution of upgrades. The engineers' multisig schedules the upgrade, while the foundation multisig executes it after the timelock expires.
+
+### Workflow
+
+1. **Deployment**:  
+   - The Contract Pipeline deploys a new contract implementation and generates transaction data for the upgrade.
+
+2. **Proposal**:  
+   - The remote key automates the submission of a transaction proposal using SafeKit API’s `proposeTransaction`, populating the transaction for review by Movement Labs engineers.
+   - The proposal includes the transaction for `timelock.schedule(proxyAdmin.upgradeAndCall(newContractImplementation))`.
+
+3. **Approval**:  
+   - The Movement Labs engineers review and gather 4/5 signatures to approve the proposal.
+   - Once approved, the `timelock.schedule` transaction is enacted, holding the upgrade in a 2-day timelock.
+
+4. **Execution**:  
+   - After the timelock period, the Movement Foundation multisig gathers 3/5 signatures to execute the upgrade.
+   - This triggers the `timelock.execute` transaction, allowing the upgrade to be applied to the ProxyAdmin, which will perform the contract upgrade.
+
+### Multisig Security Considerations
+
+- Both multisigs serve as a single signing authority, ensuring a strict separation between proposing and executing transactions.
```

Comment:
> Both multisigs serve as a single signing authority

Nit: I don't agree with this wording. They have separate functions which you described well above.